floors/enforce_test.go

```diff
@@ -662,11 +662,11 @@ func TestEnforce(t *testing.T) {
 		actEligibleBids, actErrs, actRejecteBids := Enforce(tt.args.bidRequestWrapper, tt.args.seatBids, config.Account{PriceFloors: tt.args.priceFloorsCfg}, tt.args.conversions)
 		assert.Equal(t, tt.expErrs, actErrs, tt.name)
 		if !reflect.DeepEqual(tt.expRejectedBids, actRejecteBids) {
-			assert.Fail(t, "rejected bids don't match")
+			assert.Fail(t, tt.name+": rejected bids don't match. Got: ", actRejecteBids)
 		}
 
 		if !reflect.DeepEqual(tt.expEligibleBids, actEligibleBids) {
-			assert.Fail(t, "eligible bids don't match")
+			assert.Fail(t, tt.name+": eligible bids don't match. Got: ", actEligibleBids)
```

Comment:
I agree this error message could be a bit better. How about this?
`assert.Failf(t, "eligible bids don't match", "Expected: %v, Got: %v", tt.expEligibleBids, actEligibleBids)`

Reply:
done